gradle.properties

```diff
@@ -1,20 +1,20 @@
 org.gradle.jvmargs=-Xmx1G -Dfile.encoding=UTF-8 -Duser.language=en -Duser.country=US
 
 # Fabric Properties (https://fabricmc.net/versions.html)
-## 1.19.4
-minecraft_version=1.19.4
-yarn_mappings=1.19.4+build.2
-loader_version=0.14.19
+## 1.20
+minecraft_version=1.20
+yarn_mappings=1.20+build.1
+loader_version=0.14.21
 
 #Fabric api
-## 1.19.4
-fabric_api_version=0.81.1+1.19.4
+## 1.20
+fabric_api_version=0.83.0+1.20
 
 # Dependencies
 ## Cloth Api (https://www.curseforge.com/minecraft/mc-mods/cloth-config/files)
-clothconfig_version=10.0.96
+clothconfig_version=11.0.98
 ## Mod Menu (https://www.curseforge.com/minecraft/mc-mods/modmenu/files)
-mod_menu_version=6.1.0-rc.4
+mod_menu_version=7.0.0-beta.2
 ## REI (https://www.curseforge.com/minecraft/mc-mods/roughly-enough-items/files)
 rei_version=11.0.597
```

Comment:
you forgot to change the rei_version 
1.20 uses 12.0.622

And modmenu have released full release 7.0.0